packages/aragon-client/src/index.js

```diff
@@ -91,17 +91,21 @@ export class AppProxy {
    */
   external (address, jsonInterface) {
     const contract = {
-      events: (fromBlock = 0) => {
+      events: (fromBlock) => {
+        const eventArgs = [
+          address,
+          jsonInterface.filter(
+            (item) => item.type === 'event'
+          )
+        ]
+        if (fromBlock != null) {
```

Comment:
Do we want to change this to `fromBlock !== null && fromBlock !== undefined` ? :man_shrugging:

Reply:
Hmm, actually thinking about doing a `typeof` check instead, since it should only be a number.